### ISSUE TITLE
[Request]: Rename app to comply with GNOME HIG

### ISSUE THREAD

**Issue by TheEvilSkeleton** [fabrialberio/PinApp#14]
**Is your feature request related to a problem? Please describe.**
Currently, "PinApp" uses nonstandard whitespace, which does not comply with and is discouraged by the GNOME HIG.

**Describe the solution you'd like**
I suggest renaming the app into one that complies with the GNOME HIG. One of:
- Pin Apps
- App Menu Editor (or App Menu)
- ...or a better name

**Describe alternatives you've considered**
*No response*

**Additional context**
https://developer.gnome.org/hig/guidelines/app-naming.html

**Comment by fsobolev** [comment]
> App Menu Editor (or App Menu)

This one sounds too generic, making it difficult to distinguish this particular app from other desktop files editors.

My suggestion is **Pins**.

**Comment by fabrialberio** [comment]
Yeah, it also breaks the rule about puns since a big part of why I chose "PinApp" is that it sounds a lot like pin-up. A very clever joke.

I think "Pins" is a great alternative because it fits in perfectly with the wording on buttons and other interface elements.
I would also consider something a bit more descriptive, for example "Pinboard", to give the app a bit more personality.

"Pins" is a great option in my opinion, but I am still open to suggestions and I'll reflect on it some more before making a decision.

Also, I don't intend change the app ID and the repository name, since they don't show up in the app itself.

**Comment by TheEvilSkeleton** [comment]
Pins is a great name too. And yeah, changing the app ID is entirely optional and won't affect the functionality of the app. I would suggest adding "PinApp" as a keyword in the desktop entry.

**Comment by mossymossmoss** [comment]
+1 for Pins, definitely a nicer name

**Comment by hyuri** [comment]
Pins is nice.

Some other ideas:

**Launchers**
**App Launchers**
